Frameworks/CoreText/DWriteWrapper_CoreText.mm

```diff
@@ -466,15 +466,19 @@ static CTFrameRef _DWriteGetFrame(CFAttributedStringRef string, CFRange range, C
         // Iterate through runs on the current line
         for (j = i; (j < numOfGlyphRuns) && (glyphRunDetails._baselineOriginY[j] == baselineOriginYForCurrentLine); ++j) {
             // Create _CTRun objects and make them part of _CTLine
-            _CTRun* run = [[_CTRun new] autorelease];
+            __CTRun* run = const_cast<__CTRun*>(_CTRunCreate());
+            CFAutorelease(run);
```

Comment:
really really don't prefer CFAutorelease; this is getting put into an array that will own the reference!